package.json

```diff
@@ -1,27 +1,27 @@
 {
-  "name": "url",
+  "name": "@zekkkii/url",
   "version": "1.0.0",
-  "description": "",
```

Comment:
Do note leave you project description empty. You can put the same GitHub repo description here.